.github/workflows/ci.yml

```diff
@@ -95,23 +95,21 @@ jobs:
           # `bash` needed b/c macOS ships with bash 3, which doesn't support arrays properly.
           brew install -q ninja gpg llvm@${{ matrix.clang-version }} bash z3
           echo "Z3_SYS_Z3_HEADER=/opt/homebrew/include/z3.h" >> $GITHUB_ENV
+          # It's important that we keep `RUSTFLAGS` consistent between different steps
```

Comment:
Should we also put `-D warnings` here?

Reply:
They're skipped for the `test_translator.py` step, although that step doesn't rerun cargo so it doesn't trigger a rebuild.  There's an existing comment there explaining why exactly it's skipped there (I forgot off the top of my head).

Reply:
For that step, do we still want the Z3 flags?